StreamDeckPlugin/Services/ImageService.cs

```diff
@@ -7,9 +9,19 @@
 namespace StreamDeckPlugin.Services {
     public interface IImageService {
         event Action<IDynamicActionInfo> ImageLoaded;
+        string GetImage(string imageId);
         string GetImage(IDynamicActionInfo dynamicActionInfo);
         bool HasImage(string imageId);
         void UpdateButtonImage(string name, byte[] bytes);
+
+        /// <summary>
+        /// Check to see if this image is stored- if it is not, retrieve it
+        /// </summary>
+        /// <param name="imageId">Id of the image in the cache- used to store and retrieve it</param>
+        /// <param name="cardGroupId">Card Group for the image</param>
+        /// <param name="buttonMode">ButtonMode for the image- may be null if the image is for the Card Group </param>
+        /// <param name="index">Index of the image- may be null if the image is for the Card Group</param>
+        void LoadImage(string imageId, CardGroupId cardGroupId, ButtonMode? buttonMode = null, int? index = null);
```

Comment:
Minor: for consistency, add comments to the rest of the interface

Reply:
Done- getting this into shape slowly :D